website/blog/2025-01-20-podman-quadlet.md

```diff
@@ -0,0 +1,206 @@
+
+### Why Use Quadlets?
+
+- **Declarative Configuration**: Similar to Compose or Kubernetes manifests, Quadlets allow you to declare what you want to run, simplifying the workload setup.
+- **Tight System Integration**: Quadlets align with Podman’s philosophy of integrating seamlessly with Linux, leveraging systemd’s process management capabilities.
+- **Ease of Automation**: Quadlets make it simple to configure containers to start at boot, restart on failure, and more.
+
+### Example: A Quadlet File for Nginx
+
+Below is an example of an `nginx.container` Quadlet file, which starts an nginx container at boot:
+
+```editorconfig title="~/.config/containers/systemd/nginx.container"
+# nginx.container
+[Container]
+ContainerName=nginx
+Image=nginx
+PublishPort=80:8080
+
+[Service]
+Restart=always
+```
+
+This configuration ensures the container restarts automatically if stopped, and exposes port 8080.
+
+---
+
+## Using the Podman Quadlet Extension in Podman Desktop
+
+Managing Quadlets directly on non-Linux platforms can be challenging due to virtualized environments (e.g., WSL or Hyper-V).
+Fortunately, the Podman Desktop extension Podman Quadlet simplifies this process, enabling you to list, generate, and edit Quadlets visually.
+
+### Key Features of the Extension
+
+- **Integration with Podlet**: Generates Quadlets from existing Podman objects.
+- **Quadlet Management UI**: Provides a dedicated interface to list, edit, delete, start, and stop Quadlets.
+- **Logs Viewer**: Fetches and displays systemd logs using journalctl for troubleshooting.
+
+### Installation
+
+If you already have the latest version of Podman Desktop, you can <a href="podman-desktop:extension/podman-desktop.quadlet">**click here to install the Podman Quadlet extension**</a>.
+
+Alternatively, navigate to the Extensions page within Podman Desktop to install it
+
+### Listing Quadlets :clipboard:
+
+On the Podman Quadlet page, you can view all the Quadlets available across your Podman machines. To update the list, click **Refresh**.
+
+<ThemedImage
+alt="Quadlets List"
+sources={{
+    light: require('./img/podman-quadlet/podman-quadlet-home-light.png').default,
+    dark: require('./img/podman-quadlet/podman-quadlet-home-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+In Podman Desktop, you can see the difference between containers managed by a Quadlet with a dedicated icon.
+
+<ThemedImage
+alt="Container Quadlet Icon"
+sources={{
+    light: require('./img/podman-quadlet/container-icon-quadlet-light.png').default,
+    dark: require('./img/podman-quadlet/container-icon-quadlet-dark.png').default,
+  }}
+/>
+
+### Generate Quadlets :hammer:
+
+To generate a Quadlet from an existing container, you’ll need to install [Podlet](https://github.com/containers/podlet). The extension simplifies this process:
+
+1. Go to **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > CLI Tools** and install Podlet using the Podman Quadlet extension.
```

Comment:
For better readability, maybe we can present the info in this manner:
Use one of the following ways to install Podlet:
- Go to **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > CLI Tools** and install Podlet.
- Download Podlet manually from its [GitHub release page](https://github.com/containers/podlet/releases).